BREAKING-CHANGES.md

```diff
@@ -75,7 +75,28 @@ This is a quick summary of the sections below:
   - MSRV is now 1.63.0
   - `List` no longer ignores empty strings
 
-## Unreleased (0.30.0)
+## v0.30.0 Unreleased
+
+### `Style` no longer implements `Styled` ([#1572])
+
+[#1572]: https://github.com/ratatui/ratatui/pull/1572
+
+Any calls to methods implemented by the blanket implementation of `Stylize` are now defined directly
+on `Style`. Remove the `Stylize` import if it is no longer used by your code.
+
+```diff
+- use ratatui::style::Styled;
+
+let style = Style::new().red();
+```
+
+The `reset()` method does not have a direct replacement, as it clashes with the existing `reset()`
+method. Use the `Style::reset()` method instead.
+
+```diff
+- some_style.reset();
++ Style::reset();
+```
```

Comment:
If I understood this correctly, this doesn't effect e.g. `"foo".reset()` so it should be fine.

Maybe we should clarify that? Please correct me if I'm wrong.

Reply:
There's a reset method on Stylize: `fn reset(self) -> T`
There's a reset method on Style: `fn reset() -> Style`
There's a blanket implementation of Stylize for T: Styled
Style no longer implements Styled, so no longer implements Stylize.

I'd guess that this has a low risk of impacting any real world code. It's mainly included in case it does and to set expectations that we track even minor problems like this.

Reply:
got it, I guess that is a "yes" to my initial assumption :D

Reply:
yep